hearthstone/simulator/core/hero_pool.py

```diff
@@ -512,6 +515,17 @@ def handle_event_powers(self, event: 'CardEvent', context: Union['BuyPhaseContex
 
 
 class Sindragosa(Hero):
+    base_power_cost = 1
+    power_target_location = [CardLocation.STORE]
+
+    def hero_power_valid_impl(self, context: 'BuyPhaseContext', board_index: Optional['BoardIndex'] = None,
+                              store_index: Optional['StoreIndex'] = None):
+        return bool(context.owner.store)
```

Comment:
I think we should be checking that the store index is valid further up, so this check should not be necessary.